.eslintrc.json

```diff
@@ -79,7 +74,23 @@
     "@typescript-eslint/camelcase": "warn",
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-empty-function": "off",
-    "@typescript-eslint/no-var-requires": "off"
+    "@typescript-eslint/no-var-requires": "off",
+    "react/jsx-no-bind": "off",
+    "jsx-a11y/mouse-events-have-key-events": "off",
+    "react/no-array-index-key": "off",
+    "@typescript-eslint/await-thenable": "off",
+    "@typescript-eslint/unbound-method": "off",
+    "@typescript-eslint/require-await": "off",
+    "@typescript-eslint/no-misused-promises": "off",
+    "@typescript-eslint/prefer-regexp-exec": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        "singleQuote": true,
+        "semi": false,
+        "trailingComma": "all"
+      }
+    ]
```

Comment:
Sorted the whole file with https://novicelab.org/jsonabc/

Reply:
It doesn't seem to have mattered here, but just to note for the future in case we see something weird when alphabetizing, the order of the `extends` [can matter](https://eslint.org/docs/user-guide/configuring#extending-configuration-files) in some cases.